recirq/quantum_chess/measurement_utils_test.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import cirq
+import numpy as np
+
+from recirq.quantum_chess.measurement_utils import pauli_decomposition
+
+a1 = cirq.NamedQubit('a1')
```

Comment:
Move these variables to each individual test instead of having them globally

Reply:
Thanks Victory. It's fixed.